### PR TITLE
Sync Layouts and layout styles

### LINKED ISSUE #119: Sync layouts from nitroui to neetoui
Please sync the layouts from nitroui to neetoui. 
In the previous PR i meant not to makes changes to layouts (not to delete the files) so that it can be handled in this PR.

@yedhink _a

@goutham-subramanyam Please guide yedhin on syncing layouts.

### PR DESCRIPTION
**Pull request by yedhink**
Fixes #119 

I'm not sure which layout we need to keep and which ones to discard. `Subheader` and looks like it's used in all projects, thus keeping it can be beneficial.

Please review.
@goutham-subramanyam _a

cc: @edwinbbu

### CODE REVIEW

File: lib/layouts/SubHeader.js [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+    sortProps.options.find(item => item.value === sortBy.column).label;
+
+  const showColumnFilters = R.all(R.either(R.isNil, R.isEmpty), [
+    columnFilterProps,
+    columnFilterProps?.columnFilters,
+    columnFilterProps?.handleColumnFiltering
+  ]);
+
+  const ColumnFiltersDropdown = () => {
+    const disabledColumn = "name";
```
Comment (goutham-subramanyam) [inline lib/layouts/SubHeader.js:69]: This should be removed.

File: lib/styles/layout/content-sidebar.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
+.nui-content-sidebar--wrapper {
```
Comment (goutham-subramanyam) [inline lib/styles/layout/content-sidebar.scss:1]: No need to sync this file.

File: lib/styles/layout/sidebar.scss [hunk 3]
```diff
@@ -4,13 +4,9 @@
   justify-content: flex-start;
```
Comment (goutham-subramanyam) [inline lib/styles/layout/sidebar.scss:4]: No need to sync this file.

**Review by goutham-subramanyam — CHANGES_REQUESTED** [review]
@yedhink _a

Also, please remove the badge.scss file from neetoui.

File: lib/layouts/SubHeader.js (hunk 1 above)
Comment (yedhink) [reply lib/layouts/SubHeader.js:69]: @goutham-subramanyam 

Can you clarify this a bit more? Did you mean to remove the check for `disabledColumn` all together or no need to sync this file?

**Comment by yedhink** [comment]
@goutham-subramanyam 
On removing the `badge-scss` what should be done about the `nui-badge` usage in `Badge.js` component?

**Comment by yedhink** [comment]
@goutham-subramanyam _a

Please review.

File: lib/layouts/SubHeader.js (hunk 1 above)
Comment (goutham-subramanyam) [reply lib/layouts/SubHeader.js]: The `disabledColumn` check should be removed.

File: lib/components/Badge.js [hunk 4]
```diff
@@ -7,14 +7,14 @@ const Badge = ({
...
-  onClose,
+  onClose = null,
   ...otherProps
 }) => {
   return (
     <div
       className={classnames(
         [
-          "nui-badge inline-flex items-center font-medium",
+          "inline-flex items-center font-medium",
```
Comment (goutham-subramanyam) [inline lib/components/Badge.js:17]: We need the `nui-badge` class for overriding the styles in `MenuBar`

File: lib/layouts/SubHeader.js [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+                columnFilterProps.handleColumnFiltering({
+                  name: key,
+                  checked: !column.show
+                })
+              }
+            >
+              <Checkbox
+                checked={column.show}
+                name={key}
+                label={column.label}
```
Comment (goutham-subramanyam) [inline lib/layouts/SubHeader.js:91]: We should add `onChange={() => {}}` here to avoid warnings.

**Review by goutham-subramanyam — CHANGES_REQUESTED** [review]
@yedhink _a

Please remove the imports of `badge.scss` and `sidebar.scss` from index.scss.

**Comment by yedhink** [comment]
@goutham-subramanyam _a

Please review.

File: lib/layouts/SubHeader.js [hunk 6]
```diff
@@ -0,0 +1,230 @@
...
+              key={key}
+              onClick={() =>
+                columnFilterProps.handleColumnFiltering({
+                  name: key,
+                  checked: !column.show
+                })
+              }
+            >
+              <Checkbox
+                onClick={() => {}}
```
Comment (goutham-subramanyam) [inline lib/layouts/SubHeader.js:89]: @vinay0x @edwinbbu @yedhink We have used both onClick & onChange for Checkbox in NeetoHelp. It should be consistent right? Please share your thoughts on which we should use.

References:
https://stackoverflow.com/questions/5575338/what-the-difference-between-click-and-change-on-a-checkbox
https://stackoverflow.com/questions/26615779/react-checkbox-not-sending-onchange

Comment (edwinbbu) [reply lib/layouts/SubHeader.js:89]: @goutham-subramanyam I would suggest to go with `onChange`.  

Ref
https://stackoverflow.com/questions/45617722/react-controlled-checkbox-toggle-on-props

Comment (vinay0x) [reply lib/layouts/SubHeader.js:89]: Let's use `onChange`. We'll have uniform prop naming.

Comment (yedhink) [reply lib/layouts/SubHeader.js:89]: Will make the changes.

**Comment by yedhink** [comment]
@goutham-subramanyam _a

Please review.

**Review by goutham-subramanyam — APPROVED** [review]
LGTM.

**Pull request merged**